### PR TITLE
Draft release notes for 0.7.0

### PR DESCRIPTION
**Pull request by kezhenxu94**
Thank you all for making this release possible!!! @tom-pytel @Humbertzhang @Superskyyy @cooolr @probeyang this is our final beta release and we will start to mark it 1.0.0 GA in next release cycle.

### CODE REVIEW

**Review by wu-sheng — APPROVED** [review]
Cool! We really a blog post about Python profiling. Who should be able to write this?

**Review by DeepTraLog-ICSE2022 — APPROVED** [review]

**Comment by kezhenxu94** [comment]
> Cool! We really a blog post about Python profiling. Who should be able to write this?

@Humbertzhang is the owner of this feature and I think he is the right one to write this blog

**Comment by Humbertzhang** [comment]
> > Cool! We really a blog post about Python profiling. Who should be able to write this?
> 
> @Humbertzhang is the owner of this feature and I think he is the right one to write this blog

Ok! I will write a blog about it.

**Comment by wu-sheng** [comment]
> > Cool! We really a blog post about Python profiling. Who should be able to write this?
> 
> @Humbertzhang is the owner of this feature and I think he is the right one to write this blog

Could you do that in this week? It is good perspective for this new release, and we are going to release 1.0 GA after that.

**Comment by wu-sheng** [comment]
> > Could you do that in this week? It is good perspective for this new release, and we are going to release 1.0 GA after that.
> 
> I have a paper submission deadline this Saturday night, I will try my best to finish this blog this Sunday.
> Is that ok for schedule?

Of course. You spent a lot of energy on this new feature, I don't want the community miss it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,13 +4,32 @@
...
+    - Support profiling Python method level performance (#127
     - Add a new `sw-python` CLI that enables agent non-intrusive integration (#156)
-
-- New plugins:
-    - Falcon Plugin (#146)
+    - Add exponential reconnection backoff strategy when OAP is down (#157)
+    - Suport ignoring traces by http method (#143)
+    - `NoopSpan` on queue full, propagation downstream (#141)
+    - Support agent namespace. (#126)
+    - Added `grpc.max_connection_age_grace_ms` (#130)
```
Comment (tom-pytel) [inline CHANGELOG.md:13]: This is no longer present, it was an attempt to solve what turned out to be the nginx grpc proxy problem but didn't work. Root cause was fixed later.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,13 +4,32 @@
...
+    - Support secure connection option for GRPC and HTTP (#134)
+
+- Plugins:
+    - Add Falcon Plugin (#146)
+    - Update `sw_pymongo.py` to be compatible with cluster mode (#150)
+    - Add Python celery plugin (#125)
+    - Support tornado5+ and tornado6+ (#119)
```
Comment (tom-pytel) [inline CHANGELOG.md:20]: This was when the tornado plugin was needlessly split into a v5 and v6 version even though it was the same code, it was reverted.

**Pull request merged**